outplacement_crmsync/models/res_partner.py

```diff
@@ -50,17 +69,143 @@ def get_jobseeker_data(self):
+            'phone': partner.phone,
+            'email': partner.email,
+            #
+            # outplacement_education
+            #
+            'sun_ids': [(6,0,[c.id for c in self.env['res.sun'].search([('code','in',partner.sun_ids.mapped('code'))])])],
+            'education_level': [(6,0,[e.id for e in self.env['res.partner.education_level'].search([('name','in',partner.education_level.mapped('name'))])])],
+            'foreign_education': partner.foreign_education,
+            'foreign_education_approved': partner.foreign_education_approved,
+            'cv': partner.cv,
+            'cv_file_name': partner.cv_file_name,
+            'references': partner.references,
+            'references_file_name': partner.references_file_name,
+            'has_drivers_license': partner.has_drivers_license,
+            'drivers_license_ids': [(6,0,[e.id for e in self.env['res.drivers_license'].search([('name','in',partner.drivers_license_ids.mapped('name'))])])],
+            'has_car': partner.has_car,
+            # ~ # jobs
+            'job_ids': [(6,0,[e.id for e in self.env['res.jobs'].search([('name','in',partner.job_ids.mapped('name'))])])],
+            # skills
+                # ~ skills = fields.Many2many('hr.skill', string="Skill")
+    # ~ skill_id = fields.Char(string="Skill", related="skills.complete_name")
+
+    # ~ other_experiences = fields.Many2many(comodel_name='outplacement.other_experiences', string="Other Experience")
+    # ~ strengths = fields.Many2many(comodel_name='outplacement.strengths', string="Strengths")
+    # ~ interests = fields.Many2many(comodel_name='outplacement.interests', string="Interests")
+    # ~ partner_skill_ids = fields.One2many(
+        # ~ string='Skills',
+        # ~ comodel_name='hr.employee.skill',
+        # ~ inverse_name='partner_id',
+    # ~ )
+            
+        }) 
+        
+    @api.one
+    def get_jobseeker_dataIII(self):
+        """
+        Test utan komplexa data
+        """
+        
+        xmlrpc = crm_serverII(self.env)
+        
+        
+
+        
+        partner = xmlrpc.common.env['res.partner'].browse(
+            xmlrpc.common.env['res.partner'].search([('social_sec_nr','=',self.social_sec_nr)],limit=1))
+            # ~ xmlrpc.common.env['res.partner'].search([('id','=',26)],limit=1))
+            
+        self.write({
+            'name': partner.name,
+            'street': partner.street,
+            'street2': partner.street2,
+            'zip': partner.zip,
+            'city': partner.city,
+            'phone': partner.phone,
+            'email': partner.email,
+            
+        }) 
+        
+    @api.one
+    def get_jobseeker_dataIV(self):
+        """
+        Test mot demo-data / utan partner_ssn
+        """
+        
+        xmlrpc = crm_serverII(self.env)
+        
+        
+
+        
+        partner = xmlrpc.common.env['res.partner'].browse(
+            xmlrpc.common.env['res.partner'].search([('id','=',26)],limit=1))
```

Comment:
Does this work even if there is no partner with id  = 26?

Reply:
With demo-data there Brandon Lee is 26